OCC Retries + Reset, Round 2 (#652)
usnistgov/FIPS201: _FIPS201/requirements.md

The PIN on a PIV Card may need to be reset if the cardholder has
forgotten the PIN or if PIN-based cardholder authentication has been disabled by the usage of an

> invalid PIN more than the allowed number of retries. Fingers may need to be re-enrolled for OCC if the cardholder has developed epidermal scarring or similar, resulting in false negative biometric verification decisions, or if OCC has been disabled by exceeding the allowed number of negative biometric verification decisions. No more than 10 consecutive activation retries for each of the activation methods (i.e., PIN and OCC attempts) **SHALL** be permitted. Card issuers **MAY** further restrict the maximum retry limit to a lower value.

Author: ```suggestion
invalid PIN more than the allowed number of retries. Fingers may need to be re-enrolled for OCC if the cardholder has experienced epidermal scarring or similar physical changes, resulting in false negative biometric verification decisions, or if OCC has been disabled by exceeding the allowed number of negative biometric verification decisions. No more than 10 consecutive activation retries for each of the activation methods (i.e., PIN and OCC attempts) **SHALL** be permitted. Card issuers **MAY** further restrict the maximum retry limit to a lower value.
```

> invalid PIN more than the allowed number of retries. Fingers may need to be re-enrolled for OCC if the cardholder has developed epidermal scarring or similar, resulting in false negative biometric verification decisions, or if OCC has been disabled by exceeding the allowed number of negative biometric verification decisions. No more than 10 consecutive activation retries for each of the activation methods (i.e., PIN and OCC attempts) **SHALL** be permitted. Card issuers **MAY** further restrict the maximum retry limit to a lower value.

Reviewer: ```suggestion
invalid PIN more than the allowed number of retries. Fingers might need to be re-enrolled for OCC if the cardholder has developed epidermal scarring or similar, resulting in false negative biometric verification decisions, or if OCC has been disabled by exceeding the allowed number of negative biometric verification decisions. No more than 10 consecutive activation retries for each of the activation methods (i.e., PIN and OCC attempts) **SHALL** be permitted. Card issuers **MAY** further restrict the maximum retry limit to a lower value.
```

> invalid PIN more than the allowed number of retries. Fingers may need to be re-enrolled for OCC if the cardholder has experienced epidermal scarring or similar physical changes, resulting in false negative biometric verification decisions, or if OCC has been disabled by exceeding the allowed number of negative biometric verification decisions. No more than 10 consecutive activation retries for each of the activation methods (i.e., PIN and OCC attempts) **SHALL** be permitted. Card issuers **MAY** further restrict the maximum retry limit to a lower value.

Reviewer: ```suggestion
invalid PIN more than the allowed number of retries. Fingers might need to be re-enrolled for OCC if the cardholder has experienced epidermal scarring or similar physical changes, resulting in false negative biometric verification decisions, or if OCC has been disabled by exceeding the allowed number of negative biometric verification decisions. No more than 10 consecutive activation retries for each of the activation methods (i.e., PIN and OCC attempts) **SHALL** be permitted. Card issuers **MAY** further restrict the maximum retry limit to a lower value.
```

Cardholders **MAY** change their PINs at any time by providing the current PIN and the new PIN values, as specified in [[SP 800-73]](../_Appendix/references.md#ref-SP-800-73).

> Departments and agencies **MAY** adopt more stringent procedures for PIN/OCC

Author: ```suggestion
Departments and agencies **MAY** adopt more stringent procedures for PIN or OCC
```

Regardless of the PIN reset procedure used, the chosen PIN **SHALL** meet the activation requirements specified in [Section 4.3.1](frontend.md#s-4-3-1).

#### 2.9.3.2 OCC Reset {#s-2-9-3-2}

Both fingerprints used for OCC **SHALL** be replaced during an OCC reset.

OCC reset **MAY** be performed in person at an issuing facility
or a supervised remote identity proofing station:

In person

> : When OCC reset is performed in person at the issuing facility,  before the

Author: ```suggestion
: When OCC reset is performed in person at the issuing facility, before the
```

    record.[^occ_reset_characteristic] If the biometric verification decision is
    negative or no alternative biometric data records are available, the
    cardholder **SHALL** provide the PIV Card to be reset and another primary
    identity source document (as specified in [Section
    2.7](requirements.md#s-2-7)). An attending operator **SHALL** inspect these
    and compare the cardholder with the electronic facial image retrieved from
    the enrollment data record and the photograph printed on the PIV Card.

Supervised remote identity proofing station
: PIN reset at a supervised remote identity proofing station combines the 
    assurance of an in-person reset with the convenience of a kiosk reset. All
    protections and requirements of [Section 2.7.1](requirements.md#s-2-7-1)
    **SHALL** be observed during the procedure. The operator **SHALL** initiate
    a biometric verification to ensure that the cardholder's biometric
    characteristics captured at the station elicit a positive biometric
    verification decision when compared to biometric data records stored in the
    PIV enrollment record or when compared to the biometric data records on the

>     PIV Card and using the BIO-A  authentication

Author: ```suggestion
    PIV Card and using the BIO-A authentication
```
